docs/development/maintenance.md

```diff
@@ -13,104 +12,124 @@ modules of ILIAS.
+* The people holding the **Authority to Sign of Code Changes** are responsible
+for Pull Requests to their component and get assigned related Pull Requests
+by the Technical Board according to the [Rules for Maintainers and Coordinators
+assigned to PRs](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/contributing.md#rules-for-maintainers-assigned-to-prs).
+* The person/people holding the **Authority to (De-)Assign Authorities**
+coordinate assignments of authorities with the Product Manager and the Technical
+Board, who hold a vetoing power over these decisions.
+
+# Additional Rules and Guidelines
+* Although the first decision on new features or feature removals in a unit of
+code lie with the person/people holding the **Authority to Sign of Conceptual
+Changes** the final decisions are made by them together with the product
+manager during the Jour Fixe meetings after an open discussion.
+* If nobody holds the **Authority to (De-)Assign Authorities** for a code unit
+it authomatically defaults to the Technical Board.
+* Final decision about getting write access to the ILIAS development system
+(Github) is handled by the product manager together with the Technical Board.
+* Authorities are listed with the name of the person holding the authority. In
+addition the company the person is working for can be listed, too.
+* If a company is listed for the last assignee of the **Authority to (De-)Assign
+Authorities** the company can propose a prioritized candidate for the
+succesion.
+
+## Tracking Authority
+Authorities are tracked in maintenance.json files placed in the root of the
+corresponding code unit of ILIAS. The file containes the following fields:
+
+* **"Authority to Sign of Code Changes"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users/companies on https://docu.ilias.de.
+* **"Authority to Sign of Conceptual Changes"**:
+    An array in the form [ `<username> (<userid>, <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"Authority to Sign of Testcase Changes"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"Authority to (De-)Assign Authoritiees"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"tester"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"testcase_writer"**:
+    An array in the form [ `<username> (<userid>), <company> (<company_page>)` ]
+    pointing to valid users on https://docu.ilias.de.
+* **"Unitspecific Guidelines, Rules, and Regulations"**:
+    Link to a file `COMMUNITY.md` in the root of the unity in the trunk branch on
+    Github specifying the guidelines, rules, and regulations for collaboration.
```

Comment:
Vielleicht hier auch d* Ticket-Reciever*in noch listen?

Reply:
Du meinst, die Person, die die Mantis-Tickets zugewiesen erhält? Ich weiss nicht, ob wir diese Info hier doppeln sollten. Vielleicht wollen wir dies ja auch relativ schnell einmal, z.B. bei Elternzeiten oder ähnlichem ändern können und dann wäre eine Anpassung an zwei Orten eher hinderlich, oder? Ich weiss aber nicht, wie einfach es ist, in Mantis einen Überblick über alle zugewiesenen "Ticket-Receiver:innen" zu erhalten?

Reply:
Ich stimme hier @kergomard zu, obwohl beides Vor- und Nachteile hat. Toll wäre natürlich ein Link zu Mantis, der die Personen anzeigt, die einer Komponente zugewiesen sind!